core/main.go

```diff
@@ -39,6 +41,46 @@ func main() {
 		}
 	}()
 
+	// TODO: rm this goroutine.
+	// we have it here temporarily to test systemservices pkg.
+	go func() {
```

Comment:
Should we remove this part in order the merge ?
Also I would create the systemservices instance outside of this goroutine.
Except than this it's great, I played with the influxb service in this go routine to have the memory in use and some statistics like that it's really nice